CompetitionConcepts.tex

```diff
@@ -18,23 +18,23 @@ \section{Autonomy \& Mobility}
 
 Furthermore, the specific tasks must not be solved using \emph{open loop control}.
 
-\section{Aiming for applications}
+\section{Aiming for Applications}
 \label{concept:aiming_for_applications}
-To foster advance in technology and to keep the competition interesting, the scenario and the tests will steadily increase in complexity. While in the beginning necessary abilities are being tested, tests will focus more and more on real applications with a rising level of uncertainty. Useful, robust, general, cost effective, and applicable solutions are rewarded in RoboCup@Home.
+To foster advances in technology and to keep the competition interesting, the scenario and the tests will steadily increase in complexity. While in the beginning necessary abilities are being tested, tests will focus more and more on real world applications with a rising level of uncertainty. Useful, robust, general, cost effective, and applicable solutions are rewarded in RoboCup@Home.
```

Comment:
Not sure about this change, I'd add a native speaker to triple check

Reply:
Seems good to me.